elements/lisk-transactions/test/0_transfer.ts

```diff
@@ -34,6 +34,8 @@ describe('#transfer transaction', () => {
 	const amount = '1000';
 	const fee = (0.1 * fixedPoint).toString();
 	const timeWithOffset = 38350076;
+	const networkIdentifier =
+		'e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255';
```

Comment:
It would be nice to have a function to generate the network identifier instead of hard code it everywhere.